subrootpaths.go

```diff
@@ -9,6 +9,8 @@ var (
 	srpNotPowerOf2       = errors.New("GetSubrootPaths: Supplied square size is not a power of 2")
 	srpInvalidShareCount = errors.New("GetSubrootPaths: Can't compute path for 0 share count slice")
 	srpPastSquareSize    = errors.New("GetSubrootPaths: Share slice can't be past the square size")
+
+	srpInvalidIdxStartShareCountSum = errors.New("invalid idxStart+shareCount value")
```

Comment:
[optional] if the other rename suggestion is accepted

```suggestion
	srpInvalidIdxEnd = errors.New("GetSubrootPaths: idxEnd must be larger than idxStart and shareCount")
```